cohortextractor/study_definition.py

```diff
@@ -314,6 +314,10 @@ def make_df_from_expectations(self, population):
 
             df["patient_id"] = user_df["patient_id"]
             df[colname] = user_df[returning] if returning is not None else 1
+            # The population should be the same size as the number of patients in the
+            # user-supplied CSV file. This ensures we generate no more dummy data than
+            # is necessary.
+            population = len(df)
```

Comment:
Would it be useful to log this, to provide info to the user if they tried to set an expectation_population that was different to the number of patients in the CSV file?  It might not be immediately obvious why changing the expectation_population doesn't have an effect in this case

Reply:
It would! Great suggestion. I've logged this in fabc874.